package/base-files/files/lib/upgrade/fwtool.sh

```diff
@@ -1,13 +1,9 @@
+REQUIRE_IMAGE_SIGNATURE="${REQUIRE_IMAGE_SIGNATURE:-$(uci get system.@system[-1].require_image_signature)}"
+
 fwtool_check_signature() {
 	[ $# -gt 1 ] && return 1
 
-	[ ! -x /usr/bin/ucert ] && {
-		if [ "$REQUIRE_IMAGE_SIGNATURE" = 1 ]; then
-			return 1
-		else
-			return 0
-		fi
-	}
+	[ "$REQUIRE_IMAGE_SIGNATURE" = 1 ] || return 0
```

Comment:
There are still cases where ucert might not be installed. e.g. some people might remove it. I would like to still have the check here, even the usual case would be to have ucert available.

Reply:
Once this PR is merged, then ucert is not optional anymore, it's being added to DEFAULT_PACKAGES which means it's mandatory from now on. If you remove it, fine, but then please patch fwtool as well.

Reply:
If this check is performed here without conditions, one could simplify the following lines where it is re-checked:
```
	if ! fwtool -q -s /tmp/sysupgrade.ucert "$1"; then
		echo "Image signature not found"
		[ "$FORCE" != 1 ] && {
			echo "Use sysupgrade -F to override this check when downgrading or flashing to vendor firmware"
		}
		return 1
	fi
```

As indicated in the subsequent PR, I am not sure whether return values are dealt with correctly here ...